app/jobs/search_keyword_job.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class ClientServiceError < StandardError; end
+
+class SearchKeywordJob < ApplicationJob
+  queue_as :default
```

Comment:
It is best to have a custom queue for scraping as other jobs (like sending out emails) could be on the default queue. Segregation by queus allows not block processes.